msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md

```diff
@@ -0,0 +1,158 @@
+* **Single sign-on**: You can configure your app to authenticate users with single sign-on (SSO).
+* **Languages**: You can set up or change the language of your app.
+* **Domain**: You can add the domains to load your apps in the Teams client (For example: *.example.com).
+
+:::image type="content" source="../../assets/images/tdp/configure.png" alt-text="The screenshot is an example that shows how to configure features to manage and access your app in Developer Portal.":::
+
+## Advanced
+
+In the **Advanced** section, you can see the following components to manage your app in Developer Portal:
+
+* **Owners**
+
+    Each app includes an **Owners** page, where you can share your app registration with others in your org. You can add **Administrator** and **Operative** role to manage who can change the settings of your app. The **Operative** role has the same permissions as the **Administrator** role except to delete an app.
+
+    To add an owner:
+
+    1. In the **Advanced** section, select **Owners**.
+    1. Select **Add an owner**.
+    1. Enter a name and select a user ID from the drop-down list.
+    1. Under **Role**, select **Operative** or **Administrator**.
+    1. Select **Add**.
+
+* **App content**: You can configure your app with the following additional features:
+  
+  * Loading indicator: Displays an indicator to let users know your hosted app content (For example: Tabs and Task modules) is loading.
+  * Full-screen mode: Displays a personal app without an app header. It is only supported for the published apps to your org.
+
+* **Environments**
+
+    You can configure environments and global variables to help transition your app from your local runtime to production. Global variables are used across all environments.
+
+    To set up an environment:
+
+    1. In the Developer Portal, select the **Apps** that you're working.
+    1. Go to **Environments** under **Advanced** section and select **+ Add an environment**.
+    1. Select **Add**.
+
+  * **Global variables**
+
+      1. Select **Add a global variable** to create configuration variables for your environment.
+
+      To use global variables:
+
+      Use the variable names instead of hard-coded values to set your app configurations.
+
+      1. Enter `{{` in any field in the Developer Portal. A dropdown with all the variables you've created for the chosen environment along with the global variables appears.  
+      1. Before downloading your app package (for example, when getting ready to publish to the Teams store), select the environment you want to use. Your app configurations update automatically based on the environment.
+
+* **Plan and pricing**: You can link a SaaS offer which you've created in the Partner center for your app.
+* **Admin settings**:
+  * App Customization: You can customize your app
+  * Block app by default: You can block your app by default for users until a tenant admin chooses to enable it.
+
+## Publish
```

Comment:
Publish what?

Reply:
UI component under Apps page to manage apps in developer portal, and it is explained in the start.